notes/v1.0.0-beta2.md

```diff
@@ -0,0 +1,48 @@
+# Changelog
+
+## Notable changes
+<!-- Add the list of notable changes, remove otherwise.-->
+This is a slim release on the SDK side, mainly contains bug fixes, version updates development improvements.
```

Comment:
I would perhaps say `This release contains bug fixes, version updates, and development improvements.` (Keep it positive--saying it's slim might be seen to be a negative thing. If you don't have to say it, I'd just omit.)